.vsts-ci.yml

```diff
@@ -420,7 +420,7 @@ jobs:
           R_LIB_PATH=~/Rlib
           export R_LIBS=${R_LIB_PATH}
           mkdir -p ${R_LIB_PATH}
-          RDscript -e "install.packages(c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl'),  lib = '${R_LIB_PATH}', dependencies = c('Depends', 'Imports', 'LinkingTo'), repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())" || exit 1
+          RDscript .ci/install-r-deps.R --build --test || exit 1
```

Comment:
Hmmm, I don't see `testthat` in `master` here...

Reply:
Modified this in https://github.com/microsoft/LightGBM/commit/dae117f3d94d6c49873e6759b4da1b0a8bd7ea0e, to

```shell
RDscript .ci/install-r-deps.R --build --include=RhpcBLASctl || exit 1
```